content/02.main-text.md

```diff
@@ -143,6 +144,25 @@ Styles define how references are constructed from bibliographic metadata, contro
 Thousands of journals have [predefined styles](http://editor.citationstyles.org/searchByName/).
 As a result, adopting the specific bibliographic format required by a journal usually just requires specifying the style's source URL in the Manubot configuration.
 
+To make citation-by-identifier easily usable outside of Manubot manuscripts, we created the `manubot cite` command line utility, available as a [Python package](https://pypi.org/project/manubot/).
```

Comment:
I like highlighting the new features in this paragraph, but the syntax examples break up the flow.  What are the benefits of including them here instead of as part of the Manubot package documentation?

Reply:
Agree.